apps/www/__registry__/default/ui/context-menu.tsx

```diff
@@ -92,14 +92,13 @@ ContextMenuItem.displayName = ContextMenuPrimitive.Item.displayName
 const ContextMenuCheckboxItem = React.forwardRef<
   React.ElementRef<typeof ContextMenuPrimitive.CheckboxItem>,
   React.ComponentPropsWithoutRef<typeof ContextMenuPrimitive.CheckboxItem>
->(({ className, children, checked, ...props }, ref) => (
+>(({ className, children, ...props }, ref) => (
```

Comment:
```suggestion
>(({ className, children, checked = false, ...props }, ref) => (
```
I think explicitly specifying `checked`. 
Setting the default value to `checked = false` is a good approach.

Reply:
@Kouta-fd Thanks! That's a good idea. I have adopted your suggestion!